config/database.yml

```diff
@@ -25,8 +25,9 @@ test:
 production:
   adapter: mysql2
   encoding: utf8
+  host: <%= ENV.fetch("MYSQL_HOST") { "127.0.0.1" } %>
```

Comment:
@whereismyjetpack Use `{ }` with fetch only if you need to execute the block. If you're just wanting a default value, then `ENV.fetch("MYSQL_HOST", "127.0.0.1")`  See https://ruby-doc.org/core-2.7.0/Hash.html#method-i-fetch